Lib/_pyrepl/completing_reader.py

```diff
@@ -169,7 +169,7 @@ def do(self) -> None:
         r = self.reader  # type: ignore[assignment]
         last_is_completer = r.last_command_is(self.__class__)
         immutable_completions = r.assume_immutable_completions
-        completions_unchangable = last_is_completer and immutable_completions
+        completions_unchangeable = last_is_completer and immutable_completions
         stem = r.get_stem()
         if not completions_unchangable:
```

Comment:
Won't this now result in a NameError?

Reply:
(The answer is "yes", and it's causing several tests to fail.)

You say that this PR is backporting https://github.com/python/cpython/pull/123336, but I don't see any changes to this file in https://github.com/python/cpython/pull/123336

Reply:
😨I made a mistake, I didn't notice the replacement when I made the change